atlasdb-api/src/main/java/com/palantir/atlasdb/cell/api/TransactionKeyValueServiceManager.java

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 
-package com.palantir.atlasdb.transaction.api;
+package com.palantir.atlasdb.cell.api;
```

Comment:
I don't know if this is the right package, pending your renames, but I wanted to just keep them in the same package so you can easily rename wholesale.